lib/k8s/client.rb

```diff
@@ -167,19 +167,17 @@ def resources(namespace: nil)
     # @param options @see K8s::ResourceClient#list
     # @return [Array<K8s::Resource>]
     def list_resources(resources = nil, **options)
-      cached_clients = @api_clients.size > 0
+      cached_clients = @api_clients.size.positive?
```

Comment:
Comment got removed because of refactor, should this be `||=` to avoid an infinite retry loop?

Reply:
When api_groups! adds something to @api_clients

Reply:
Oh, but it's outside of begin/rescue, nvm